projects/plugins/boost/app/assets/src/js/features/page-cache/meta/meta.tsx

```diff
@@ -1,19 +1,69 @@
-import { Button } from '@automattic/jetpack-components';
-import { __ } from '@wordpress/i18n';
+import { Button, Notice } from '@automattic/jetpack-components';
+import { createInterpolateElement } from '@wordpress/element';
+import { __, _n, sprintf } from '@wordpress/i18n';
 import ChevronDown from '$svg/chevron-down';
 import ChevronUp from '$svg/chevron-up';
 import Lightning from '$svg/lightning';
 import styles from './meta.module.scss';
-import { useState } from 'react';
+import { useEffect, useState } from 'react';
+import { usePageCache } from '$lib/stores/page-cache';
+import { Link } from 'react-router-dom';
+import classNames from 'classnames';
 
 const Meta = () => {
 	const [ isExpanded, setIsExpanded ] = useState( false );
+	const [ query, mutation ] = usePageCache();
```

Comment:
Looks like `usePageCache` might be using DataSync. Should this be wrapped in an Error Boundary? ( https://wp.me/pc9hqz-2wT ).

Reply:
Thanks! Updated.